packages/expo/src/launch/registerRootComponent.expo.tsx

```diff
@@ -1,15 +1,30 @@
-import 'expo/build/Expo.fx';
+import Constants from 'expo-constants';
 import { activateKeepAwake } from 'expo-keep-awake';
 import * as React from 'react';
-import { AppRegistry } from 'react-native';
+import { AppRegistry, YellowBox } from 'react-native';
 
 import withExpoRoot from './withExpoRoot';
 import { InitialProps } from './withExpoRoot.types';
 
 if (__DEV__) {
+  if (Constants.manifest?.experiments?.redesignedLogBox) {
```

Comment:
perhaps unstable or dangerous?
```suggestion
  if (Constants.manifest?.dangerous?.enableLogBox) {
```